packages/application-shell/src/components/quick-access/butler-command/butler-command.tsx

```diff
@@ -1,13 +1,22 @@
-import React from 'react';
+import React, { MouseEventHandler } from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import { AngleThinRightIcon } from '@commercetools-uikit/icons';
 import { customProperties } from '@commercetools-uikit/design-system';
+import { Command } from '../types';
 
-const ButlerCommand = ({ command, onMouseEnter, onClick, isSelected }) => (
+type Props = {
+  command: Command;
+  isSelected?: boolean;
+  onClick: MouseEventHandler<HTMLDivElement>;
+  onMouseEnter: MouseEventHandler<HTMLDivElement>;
+};
+
+const ButlerCommand = (props: Props) => (
   <div
-    key={command.id}
-    data-testid={`quick-access-result(${command.id})`}
+    key={props.command.id}
+    data-testid={`quick-access-result(${props.command.id})`}
+    aria-current={props.isSelected === true ? 'true' : 'false'}
```

Comment:
This helps in the tests to make some assertions (previously css colors were used)